### PR TITLE
Fix: Dynamic texts for CHECK constraint placeholder

### LINKED ISSUE #28590: The placeholder in "CHECK Constraint" input is not suggesting the correct example
# Bug report

- [x] I confirm this is a bug with Supabase, not with my own application. 
- [x] I confirm I have searched the [Docs](https://docs.supabase.com), GitHub [Discussions](https://github.com/supabase/supabase/discussions), and [Discord](https://discord.supabase.com). 

## Describe the bug

I created a table with the `duration_min` field as `int4`.
This is what I see in the check constraint
image

## To Reproduce

Steps to reproduce the behavior, please provide code snippets or a repository:

1. Create a table which include a `duration_min` field. Set `duration_min` to `int4`.
2. Click save. So the table is created now.
3. Click to view the newly created table.
4. Click on the dropdown menu of the `duration_min` column
5. Click "Edit Column"
6. Scroll down and see
image
7. Follow the suggestion will lead to error.

## Expected behavior

Show `e.g duration_min > 0` or something correct.

## System information

- OS: Ubuntu
- Browser Chrome

## Additional context

I use supabase.com in the browser.

### PR DESCRIPTION
**Pull request by major-winter**
## I have read the [CONTRIBUTING.md](https://github.com/supabase/supabase/blob/master/CONTRIBUTING.md) file.

YES

## What kind of change does this PR introduce?

Bug fix

## What is the current behavior?

The current behavior is described in this [bug item](https://github.com/supabase/supabase/issues/28590)

## What is the new behavior?

Placeholder for CHECK constraint is now dynamic depending on the selected Postgres Data Type

## Additional context
jsonb
image

### CODE REVIEW

File: apps/studio/components/interfaces/TableGridEditor/SidePanelEditor/ColumnEditor/ColumnEditor.tsx [hunk 1]
```diff
@@ -133,7 +134,52 @@ const ColumnEditor = ({
...
+      case "float8":
+        setPlaceholder(`${columnName} > 0.0`)
+        break
+      case "text":
+      case "varchar":
+        setPlaceholder(`LENGTH(${columnName}) <= 50`)
+        break;
+      case "json":
+      case "jsonb":
+        setPlaceholder(`JSONB_TYPEOF(settings->'active') = 'boolean'`)
```
Comment (alaister) [inline apps/studio/components/interfaces/TableGridEditor/SidePanelEditor/ColumnEditor/ColumnEditor.tsx:156]: Should this be something like:
```suggestion
        setPlaceholder(`JSONB_TYPEOF(${columnName}->'active') = 'boolean'`)
```
?

Comment (major-winter) [reply apps/studio/components/interfaces/TableGridEditor/SidePanelEditor/ColumnEditor/ColumnEditor.tsx]: You're right, I've updated this field.

**Comment by Hallidayo** [comment]
@supabase/frontend

**Review by alaister — APPROVED** [review]
Thanks @major-winter and sorry about the delay here!

**Pull request merged**